### PR TITLE
Feat/implement view current profile information and picture in input fields

### PR DESCRIPTION
**Pull request by SirhmVFX**
# Description
View Current Profile Information and Picture in Input Fields
Implement UI for Uploading and changing Profile Picture
Implement UI for Editing Profile Information with Validation
Implement Delete Profile picture

# Link to ticket
https://linear.app/team-astrid-zara-orji/issue/BFF-4/implement-ui-for-editing-profile-information-with-validation
https://linear.app/team-astrid-zara-orji/issue/BFF-3/implement-ui-for-uploading-and-changing-profile-picture
View Current Profile Information and Picture in Input Fields

**Closes #issue_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **dev branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

**Review by phoenixdahdev — DISMISSED** [review]

File: src/config/auth.config.ts [hunk 1]
```diff
@@ -150,6 +150,7 @@ export default {
     signIn: "/login",
     error: "/error",
   },
-  secret: process.env.AUTH_SECRET,
+  secret:
+    process.env.AUTH_SECRET || "IBq7Jk+ZYYvcxMWQQfsDCTC3Q0qYO+ukk0wyYRKuSqk=",
```
Comment (prudentbird) [inline src/config/auth.config.ts]: Why are you hardcoding an auth secret?

**Review by prudentbird — CHANGES_REQUESTED** [review]
Remove all comments

File: src/config/auth.config.ts (hunk 1 above)
Comment (SirhmVFX) [reply src/config/auth.config.ts]: i didn't do anything around auth, probably from my last pull, but would fixx it sir

File: .env.sample
Comment (prudentbird) [inline .env.sample:1]: Why did you delete this file 🫠👀

**Review by prudentbird — CHANGES_REQUESTED** [review]

**Comment by prudentbird** [comment]
Just only make changes to files relating to your feature

**Review by prudentbird — DISMISSED** [review]

**Review by prudentbird — DISMISSED** [review]

**Review by phoenixdahdev — DISMISSED** [review]

**Review by prudentbird — APPROVED** [review]

**Review by phoenixdahdev — APPROVED** [review]
